### PR TITLE
chore: Update gradle and readme with saucelabs instruction and known issue

### PR DESCRIPTION
**Pull request by sweeneyapps**
this closes: #12

- Moved ADT API key to Gradle variable
- Added saucelabs instruction in the README
- Update gradle plugin to 7.3.1
- Added known issue to README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,23 +2,40 @@
...
-        }
-    }
+android {
+    def AXE_DEVTOOLS_APIKEY = "YOUR_API_KEY_HERE"
+    ...
+}
 ```
 
-Once you have a working API Key installed, you can run the `ExampleEndToEndAccessibilityTest` Espresso test manually, or through automated services such as SauceLabs. This repository has a prewritten `.sauce/config.yml`, so if you have [saucectl](https://github.com/saucelabs/saucectl) installed, run `saucectl run` and you will see the Sample app end-to-end test on your SauceLabs account! 
+Once you have a working API Key installed, you can run the app with manual testing FAB or run the `ExampleEndToEndAccessibilityTest` Espresso test manually, or through automated services such as SauceLabs.
```
Comment (karnett) [inline README.md:14]:
```suggestion
Once you add the API Key, you can run the app. The app is ready to go for accessibility testing by tapping on the floating action button to kick off a scan at any point you'd like. <!-- Insert documentation link once new docs are updated -->.

You can also see accessibility testing in action through the  `ExampleEndToEndAccessibilityTest` Espresso. You can kick it off manually, or through an automated service such as SauceLabs.
```

File: README.md [hunk 2]
```diff
@@ -2,23 +2,40 @@
...
 
+
+## SauceLabs
+
+If you have account with Saucelabs, you can run automated test with this app by using our prewritten SauceLabs configuration file `.sauce/config.yml` on this repository and follow the steps below:
```
Comment (karnett) [inline README.md:19]:
```suggestion
If you have an account with Saucelabs, you can run the automated tests of this application by using our prewritten SauceLabs configuration file `.sauce/config.yml`. Follow the steps below to get started:
```

File: README.md [hunk 3]
```diff
@@ -2,23 +2,40 @@
...
+
+1. Install saucectl command-line interface. [See Using the saucectl CLI](https://docs.saucelabs.com/dev/cli/saucectl/)
+
+1. Add your Sauce Labs credentials to your .bash_profile or .zshenv. Be sure to load the changes by running source .filename. [See Associate your Credentials](https://docs.saucelabs.com/dev/cli/saucectl/#associate-your-credentials)
+
+1. Update the prewritten `.sauce/config.yml` [See Configuring Your Espresso Tests](https://docs.saucelabs.com/mobile-apps/automated-testing/espresso-xcuitest/espresso/)
+
+1. Run test on Saucelabs from terminal: `./gradlew app:assembleDebug && ./gradlew app:assembleAndroidTest && ./saucectl run`
```
Comment (karnett) [inline README.md:27]:
```suggestion
1. Run test on Sauce Labs from terminal: `./gradlew app:assembleDebug && ./gradlew app:assembleAndroidTest && ./saucectl run`
```

File: README.md [hunk 4]
```diff
@@ -2,23 +2,40 @@
...
+
+
+## Known Issue
+
+There is an issue running the app on JDK higher than 11.
+
+The gradle plugin is updated with Android Studio Dolphin but if you are using older Android Studio, you can update the gradle plugin to make it work.
```
Comment (karnett) [inline README.md:34]:
```suggestion
The gradle plugin is updated with Android Studio Dolphin. If you are using an older version of Android Studio, update the gradle plugin to make it work.
```
Can we add a link of instructions to update the gradle plugin?

**Review by devchan4188 — APPROVED** [review]
LGTM!

**Review by karnett — APPROVED** [review]

**Pull request merged**